packages/terra-functional-testing/src/commands/expect/toMatchReference.js

```diff
@@ -35,8 +36,18 @@ function toMatchReference(screenshot) {
     message = message.concat(`Expected the screenshot to be within the mismatch tolerance, but received a mismatch difference of ${misMatchPercentage}%.`);
   }
 
+  let pass = global.Terra.serviceOptions.ignoreScreenshotMismatch || imagesMatch;
```

Comment:
We should do all of these only if [useRemoteReferenceScreenshots](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/terra-cli/wdio/index.js#L187-L191) option that was added on PR #732 was set to true. We should skip all of this if a consumer isn't using this feature.

Reply:
Added a check here https://github.com/cerner/terra-toolkit/pull/736/commits/e3ea3399352b384b0e54bca1fecb239726424110#diff-1372ed3653dff72eae9a8aa8b3969b4e50473f645766c7c2f232298e514d0518R41